manage-config

```diff
@@ -108,6 +108,7 @@ if [ -e ${exclusion_file} -o -e ${inclusion_file} ]; then
             if [ ! -z "$opt" ] && [[ ! "$opt" =~ ^#.* ]]; then
                 n=${opt%=*}
                 v="${opt#*=}"
+                v=$(sed -e 's/^"//' -e 's/"$//' <<< "$v")
```

Comment:
As Bash’s [parameter expansion](https://manpages.debian.org/buster/bash/bash.1.en.html#Parameter_Expansion) is already used above, could you please also do this, instead of using *sed*?

Reply:
Sure, done it.